src/gtest/test_noteencryption.cpp

```diff
@@ -117,15 +118,15 @@ uint256 test_prf(
 TEST(noteencryption, prf_addr)
 {
     for (size_t i = 0; i < 100; i++) {
-        uint256 a_sk = libzcash::random_uint256();
+        uint252 a_sk = uint252(libzcash::random_uint256());
```

Comment:
If this were a separate ``random_uint252`` function, it would allow the constructor to throw when given an invalid input (not zeroes in the first four bits).